scalding-core/src/main/scala/com/twitter/scalding/ExecutionContext.scala

```diff
@@ -31,6 +36,25 @@ trait ExecutionContext {
   def flowDef: FlowDef
   def mode: Mode
 
+  def getIdentifierOpt(descriptions: Seq[String]): Option[String] = {
+    if (descriptions.nonEmpty) {
+      Some(descriptions.distinct.mkString(", "))
```

Comment:
agreed to make it private.  I wanted the sorting to be in order and did the distinct just to be safe.